etl-pipeline/tests/functional/processes/ingest/assert_uploaded_epubs.py

```diff
@@ -0,0 +1,14 @@
+from model import Record
+from managers.s3 import S3Manager
+
+
+def assert_uploaded_epubs(records: list[Record]):
+    s3_manager = S3Manager()
+
+    for record in records:
+        manifest_part = next((part for part in record.parts if part.file_type == 'application/epub+zip'), None)
+
+        if manifest_part and 'epub' in manifest_part.url:
+            manifest_head_response = s3_manager.client.head_object(Key=manifest_part.file_key, Bucket=manifest_part.file_bucket)
+
+            assert manifest_head_response is not None
```

Comment:
Are there cases where epub is not in the url?

Reply:
No there shouldn't be because in the LOCMapping when we create a Part with a file type of application/epub+zip the url always has the string 'epub' at the end of it due to the epub_location variable.

```
if 'epub_file' in first_file_resource.keys():
        record_id = source_id.split('|')[0]
        epub_location = f'epubs/loc/{record_id}.epub'
        epub_url = get_stored_file_url(storage_name=os.environ['FILE_BUCKET'], file_path=epub_location)

        has_part.append(str(Part(
            index=1,
            url=epub_url,
            source=Source.LOC.value,
            file_type='application/epub+zip',
            flags=str(FileFlags(download=True)),
            source_url=first_file_resource['epub_file']
        )))
```